Add automation for first two CAT APIs (#8930)
opensearch-project/documentation-website: _api-reference/cat/cat-allocation.md

The CAT allocation operation lists the allocation of disk space for indexes and the number of shards on each node.

<!-- spec_insert_start
api: cat.allocation
component: paths_and_http_methods
-->
## Paths and HTTP methods
```json
GET /_cat/allocation
GET /_cat/allocation/{node_id}
```
<!-- spec_insert_end -->

<!-- spec_insert_start
api: cat.allocation
component: query_parameters
-->
## Query parameters
Parameter | Type | Description

> :--- | :--- | :---

Reviewer: We also need to specify defaults for the query parameters that have them (e.g., `local`).

> `bytes` | String | The unit used to display byte values.

Reviewer: ```suggestion
`bytes` | String | The units used to display byte values.
```

> `cluster_manager_timeout` | String | Operation timeout for connection to cluster-manager node.

Reviewer: ```suggestion
`cluster_manager_timeout` | String | A timeout for connection to the cluster manager node.
```

> `format` | String | A short version of the Accept header (for example, `json`, `yaml`).

Reviewer: ```suggestion
`format` | String | A short version of the HTTP accept header (for example, `json` or `yaml`).
```

> `h` | List | Comma-separated list of column names to display.

Reviewer: ```suggestion
`h` | List | A comma-separated list of column names to display.
```

> `help` | Boolean | Return help information.

Reviewer: ```suggestion
`help` | Boolean | Whether to return help information.
```

> `local` | Boolean | Return local information, do not retrieve the state from cluster-manager node.

Reviewer: ```suggestion
`local` | Boolean | If `true`, specifies to retrieve information only from the local node. If `false`, specifies to retrieve local information from the cluster manager node.
```

> `s` | List | Comma-separated list of column names or column aliases to sort by.

Reviewer: ```suggestion
`s` | List | A comma-separated list of column names or column aliases to sort the response.
```

> `v` | Boolean | Verbose mode. Display column headers.

Reviewer: ```suggestion
`v` | Boolean | Whether to display column headers.
```

The CAT allocation operation lists the allocation of disk space for indexes and the number of shards on each node.

<!-- spec_insert_start
api: cat.allocation
component: endpoints
-->
## Endpoints
```json
GET /_cat/allocation
GET /_cat/allocation/{node_id}
```
<!-- spec_insert_end -->

<!-- spec_insert_start
api: cat.allocation
component: query_parameters
columns: Parameter,Type,Description,Default
include_deprecated: false
-->
## Query parameters
Parameter | Type | Description | Default
:--- | :--- | :--- | :---
`bytes` | String | The units used to display byte values. | 
`cluster_manager_timeout` | String | A timeout for connection to the cluster manager node. | 
`format` | String | A short version of the HTTP `Accept` header, such as `json` or `yaml`. | 
`h` | List | A comma-separated list of column names to display. | 
`help` | Boolean | Returns help information. | `false`

> `local` | Boolean | Returns local information but does not retrieve the state from cluster-manager node. | `false`

Reviewer: ```suggestion
`local` | Boolean | Returns local information but does not retrieve the state from cluster manager node. | `false`
```
